tokio/src/runtime/driver.rs

```diff
@@ -73,10 +74,37 @@ cfg_not_signal_internal! {
     }
 }
 
+// ===== process driver =====
+
+cfg_process_driver! {
+    type ProcessDriver = crate::park::Either<crate::process::unix::driver::Driver, SignalDriver>;
```

Comment:
This does not have to happen in this PR. I believe that, instead of defining an `Either` for each individual component (I/O, signal, process), you *probably* can have a single `Either` for the entire "I/O driver stack". This is because, at runtime, you can either enable I/O, signal, and process or disable all of them.

Reply:
Good idea, this was all left over from before we decided to group the I/O stack together. I'll have to think a bit about how to implement this and I'll open a follow up PR